bookmarks/views/settings.py

```diff
@@ -95,3 +102,11 @@ def _find_message_with_tag(messages, tag):
     for message in messages:
         if message.extra_tags == tag:
             return message
+
+def _get_app_version():
```

Comment:
We can optimize this to run only once. Instead of using a function, I think we could put this into the scope of the module and store the result in a `app_version` variable. So something like:
```python
...

try:
    with open("version.txt", "r") as f:
        app_version = f.read().strip("\n")
except Exception as exc: 
    logging.exception(exc)
    pass

#... Remaining code in the code

@login_required
def general(request):
    if request.method == 'POST':

```